README.md

```diff
@@ -2,12 +2,19 @@
 
 This is a set of Gradle build scripts to generate code for `Adyen/adyen-*-api-library` repositories. 
 
-To generate all services in all libraries:
+To generate all services (see [`adyen.sdk-automation-conventions.gradle`](/buildSrc/src/main/groovy/adyen.sdk-automation-conventions.gradle)) in all libraries:
+
+> *Note:* The following models/services are not generated for Platform & Financial Services (legacy VIAS):
```

Comment:
Not sure we should keep a list for this . I think a reference to the conventions file would do.

Reply:
Fixed, added the link